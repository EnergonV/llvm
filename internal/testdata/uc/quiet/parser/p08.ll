; ModuleID = 'p08.c'
target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"

; Function Attrs: nounwind uwtable
define i32 @main() #0 {
  %1 = alloca i32, align 4
  %x = alloca i32, align 4
  %y = alloca i32, align 4
  store i32 0, i32* %1
  %2 = load i32, i32* %x, align 4
  %3 = icmp ne i32 %2, 0
  br i1 %3, label %4, label %10

; <label>:4                                       ; preds = %0
  %5 = load i32, i32* %y, align 4
  %6 = icmp ne i32 %5, 0
  br i1 %6, label %7, label %8

; <label>:7                                       ; preds = %4
  store i32 4711, i32* %x, align 4
  br label %9

; <label>:8                                       ; preds = %4
  store i32 42, i32* %x, align 4
  br label %9

; <label>:9                                       ; preds = %8, %7
  br label %10

; <label>:10                                      ; preds = %9, %0
  %11 = load i32, i32* %1
  ret i32 %11
}

attributes #0 = { nounwind uwtable "disable-tail-calls"="false" "less-precise-fpmad"="false" "no-frame-pointer-elim"="true" "no-frame-pointer-elim-non-leaf" "no-infs-fp-math"="false" "no-nans-fp-math"="false" "stack-protector-buffer-size"="8" "target-cpu"="x86-64" "target-features"="+sse,+sse2" "unsafe-fp-math"="false" "use-soft-float"="false" }

!llvm.ident = !{!0}

!0 = !{!"clang version 3.7.1 (tags/RELEASE_371/final)"}
