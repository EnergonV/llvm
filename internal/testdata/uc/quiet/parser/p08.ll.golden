target datalayout = "e-m:e-i64:64-f80:128-n8:16:32:64-S128"
target triple = "x86_64-unknown-linux-gnu"
define i32 @main() {
0:
	%1 = alloca i32
	%x = alloca i32
	%y = alloca i32
	store i32 0, i32* %1
	%2 = load i32, i32* %x
	%3 = icmp ne i32 %2, 0
	br i1 %3, label %4, label %10
4:
	%5 = load i32, i32* %y
	%6 = icmp ne i32 %5, 0
	br i1 %6, label %7, label %8
7:
	store i32 4711, i32* %x
	br label %9
8:
	store i32 42, i32* %x
	br label %9
9:
	br label %10
10:
	%11 = load i32, i32* %1
	ret i32 %11
}
